confluent_kafka/kafkatest/verifiable_consumer.py

```diff
@@ -178,11 +169,11 @@ def msg_consume (self, msg):
             return
 
         if False:
-            self.dbg('Read msg from %s [%d] @ %d' % \
+            self.dbg('Read msg from %s [%d] @ %d' %
                      (msg.topic(), msg.partition(), msg.offset()))
 
         if self.max_msgs >= 0 and self.consumed_msgs >= self.max_msgs:
-            return # ignore extra messages
+            return  # ignore extra messages
```

Comment:
yeah, inline comments are supposed to get 2 spaces. We can choose some warnings to disable if they get too annoying.